frontend/src/features/Dashboard/components/DashboardForm/Accordion.tsx

```diff
@@ -0,0 +1,67 @@
+import { Accent, Icon, IconButton, Size } from '@mtes-mct/monitor-ui'
+import styled from 'styled-components'
+
+type AccordionProps = {
+  children: React.ReactNode
+  headerButton?: React.ReactNode
+  isExpanded: boolean
+  setExpandedAccordion: () => void
+  title: string
+}
+
+export function Accordion({ children, headerButton, isExpanded, setExpandedAccordion, title }: AccordionProps) {
+  return (
+    <AccordionContainer $withCursor={!headerButton}>
+      <AccordionHeader onClick={!headerButton ? setExpandedAccordion : undefined}>
+        <TitleContainer>
+          <Title>{title}</Title>
+          {headerButton}
+        </TitleContainer>
+        <StyledIconButton
+          $isExpanded={isExpanded}
+          accent={Accent.TERTIARY}
+          Icon={Icon.Chevron}
+          onClick={setExpandedAccordion}
+          size={Size.SMALL}
+        />
+      </AccordionHeader>
+      <HeaderSeparator />
+      <AccordionContent $isExpanded={isExpanded}>{children}</AccordionContent>
+    </AccordionContainer>
+  )
+}
+
+const AccordionContainer = styled.div<{ $withCursor: boolean }>`
+  box-shadow: 0px 3px 6px #70778540;
+  cursor: ${({ $withCursor }) => ($withCursor ? 'pointer' : 'default')};
+`
+const StyledIconButton = styled(IconButton)<{ $isExpanded: boolean }>`
+  transform: ${({ $isExpanded }) => ($isExpanded ? 'rotate(180deg)' : 'rotate(0deg)')};
+  transition: transform 0.3s;
+`
+const AccordionHeader = styled.div`
+  display: flex;
+  justify-content: space-between;
+  padding: 24px;
+`
+const TitleContainer = styled.div`
+  align-items: center;
+  display: flex;
+  gap: 16px;
+`
+const Title = styled.span`
```

Comment:
suggestion: on devrait etre sur du h2 si le titre de l'onglet est h1
```suggestion
const Title = styled.h2`
```